models/ed/R/other.helpers.ED2.R

```diff
@@ -2,9 +2,33 @@
 #' 
 #' @author Alexey Shiklomanov
 #' @inheritParams base::list.files
+#' @export
```

Comment:
is this  a function that is needed outside of the ED code?

Reply:
not currently, but it could be